hack/config.yaml

```diff
@@ -11,6 +11,9 @@ rapl:
 
 enable-pprof: true # Enable pprof debug endpoints
 
+web:
+  configFile: "" # Path to TLS server config file
```

Comment:
the default value is already empty string.

Reply:
we could add a separate sample which has a defined web config also, and refers to that file in this config.yaml